docs/CMakeLists.txt

```diff
@@ -57,7 +61,7 @@ ExternalProject_Add(docutils
 	CONFIGURE_COMMAND ""
     BUILD_IN_SOURCE 1
     BUILD_COMMAND ${PYTHON_CMD} setup.py build
-    INSTALL_COMMAND ${PYTHON_CMD} setup.py install --prefix=${EXTDIST_ROOT}
+    INSTALL_COMMAND ${PYTHON_CMD} setup.py install --prefix=${EXTDIST_ROOT} --install-lib=${EXTDIST_PYTHONPATH}
```

Comment:
EXTIDST_PYTHONPATH is from the branch RB-1.1.  The master branch does not have it now.

Reply:
Why `--install-lib` is needed? 
i.e. `--prefix` already provides the root installation path.

Reply:
Without setting it I get the following (with 1.1.1):
```
[ 71%] Performing install step for 'Pygments'
cd /builddir/build/BUILD/OpenColorIO-1.1.1/build/docs/Pygments-prefix/src/Pygments && PYTHONPATH=/builddir/build/BUILD/OpenColorIO-1.1.1/build/ext/dist/lib64/python3.7/site-packages: /usr/bin/python3 setup.py install --prefix=/builddir/build/BUILD/OpenColorIO-1.1.1/build/ext/dist
running install
Checking .pth file support in /builddir/build/BUILD/OpenColorIO-1.1.1/build/ext/dist/lib/python3.7/site-packages/
/usr/bin/python3 -E -c pass
make[2]: Leaving directory '/builddir/build/BUILD/OpenColorIO-1.1.1/build'
TEST FAILED: /builddir/build/BUILD/OpenColorIO-1.1.1/build/ext/dist/lib/python3.7/site-packages/ does NOT support .pth files
error: bad install directory or PYTHONPATH

You are attempting to install a package to a directory that is not
on PYTHONPATH and which Python does not read ".pth" files from.  The
installation directory you specified (via --install-dir, --prefix, or
the distutils default setting) was:

    /builddir/build/BUILD/OpenColorIO-1.1.1/build/ext/dist/lib/python3.7/site-packages/

and your PYTHONPATH environment variable currently contains:

    '/builddir/build/BUILD/OpenColorIO-1.1.1/build/ext/dist/lib64/python3.7/site-packages:'

Here are some of your options for correcting the problem:

* You can choose a different installation directory, i.e., one that is
  on PYTHONPATH or supports .pth files

* You can add the installation directory to the PYTHONPATH environment
  variable.  (It must then also be on PYTHONPATH whenever you run
  Python and want to use the package(s) you are installing.)

* You can set up the installation directory to support ".pth" files by
  using one of the approaches described here:

  https://setuptools.readthedocs.io/en/latest/easy_install.html#custom-installation-locations


Please make the appropriate changes for your system and try again.
```

Reply:
Please keep in mind that OCIO `master` (i.e. v2) does not support Python 3 so the compilation is undefined for now.

Reply:
For the record, I did successful compilations on ubuntu 14, 16 and 18 versions and CentOS 6 with Python 2.7. So, I would first investigate a Python 3.x difference.

Reply:
I've got PYGLUE turned off as there are not any consumers of the python module at this time. I guess I could just use python2 then but I don't think that would address the above error.

Reply:
Ok, slightly different but still fails:
```
[ 12%] Performing build step for 'Jinja2'
cd /builddir/build/BUILD/OpenColorIO-1.1.1/build/docs/Jinja2-prefix/src/Jinja2 && PYTHONPATH=/builddir/build/BUILD/OpenColorIO-1.1.1/build/ext/dist/lib64/python2.7/site-packages: /usr/bin/python2 setup.py build
BUILDSTDERR: Traceback (most recent call last):
BUILDSTDERR:   File "setup.py", line 40, in <module>
BUILDSTDERR:     from setuptools import setup
BUILDSTDERR: ImportError: No module named setuptools
BUILDSTDERR: make[2]: *** [docs/CMakeFiles/Jinja2.dir/build.make:117: docs/Jinja2-prefix/src/Jinja2-stamp/Jinja2-build] Error 1
```

Reply:
When building on ubuntu 18 with Python 2.7, I found the information to compare with your build.
`make docs`

Here are the first lines for `Jinja`:
```Scanning dependencies of target Jinja2
[  6%] Creating directories for 'Jinja2'
[  6%] Performing download step (verify and extract) for 'Jinja2'
-- extracting...
     src='/home/devel/git/adsk_priv_ocio/ext/Jinja2-2.7.1.tar.gz'
     dst='/home/devel/git/adsk_priv_ocio/build/docs/Jinja2-prefix/src/Jinja2'
-- extracting... [tar xfz]
-- extracting... [analysis]
-- extracting... [rename]
-- extracting... [clean up]
-- extracting... done
[  7%] No patch step for 'Jinja2'
[  7%] No update step for 'Jinja2'
[  7%] No configure step for 'Jinja2'
[  7%] Performing build step for 'Jinja2'
running build
running build_py
creating build
creating build/lib.linux-x86_64-2.7
creating build/lib.linux-x86_64-2.7/jinja2
```

Here are the generated lines in `build.make` line 117:

> BUILDSTDERR: make[2]: *** [docs/CMakeFiles/Jinja2.dir/build.make:117: docs/Jinja2-prefix/src/Jinja2-stamp/Jinja2-build] Error 1

```
cd /home/devel/git/adsk_priv_ocio/build/docs/Jinja2-prefix/src/Jinja2 && PYTHONPATH=/home/devel/git/adsk_priv_ocio/build/ext/dist/lib/python2.7/site-packages /usr/bin/python2.7 setup.py build
cd /home/devel/git/adsk_priv_ocio/build/docs/Jinja2-prefix/src/Jinja2 && /usr/bin/cmake -E touch /home/devel/git/adsk_priv_ocio/build/docs/Jinja2-prefix/src/Jinja2-stamp/Jinja2-build
```

My guess is that `setuptools` was not installed. Below are the first lines of the build when doing `make docs`:

```
Scanning dependencies of target pythonpath_create
[  0%] Built target pythonpath_create
Scanning dependencies of target setuptools
[  0%] Creating directories for 'setuptools'
[  0%] Performing download step (verify and extract) for 'setuptools'
-- extracting...
     src='/home/devel/git/adsk_priv_ocio/ext/setuptools-1.1.6.tar.gz'
     dst='/home/devel/git/adsk_priv_ocio/build/docs/setuptools-prefix/src/setuptools'
-- extracting... [tar xfz]
-- extracting... [analysis]
-- extracting... [rename]
-- extracting... [clean up]
-- extracting... done
[  1%] No patch step for 'setuptools'
[  1%] No update step for 'setuptools'
[  1%] No configure step for 'setuptools'
[  1%] Performing build step for 'setuptools'
running build
running build_py
creating build
creating build/lib.linux-x86_64-2.7
```
Please refer to `[]\docs\CMakeLists.txt` from line 35 to investigate. More specifically I would check the content of `PYTHONPATH`

Here is the content of `PYTHONPATH` for my build:
`/home/devel/git/adsk_priv_ocio/build/ext/dist/lib/python2.7/site-packages`


Once again (i.e. compilations with Python 3.x and Python 2.x), the `PYTHONPATH` content is suspicious.